public-html/aboutus.php

```diff
@@ -1,75 +1,57 @@
-<?php 
-require_once($_SERVER["DOCUMENT_ROOT"] . "/../vendor/autoload.php");
+<?php
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
 
-use src\SessionCookie;
-use template\Menu;
+use template\Page;
 
-$session = new SessionCookie;
-
-?>
-<html>
-<title>About us</title>
-
-<head>
-    <?php
-    if (isset($_SESSION["logged_in"])) {
-        $menu = new Menu(basename(__FILE__), $_SESSION["admin"], $_SESSION["uname"]);
-    } else {
-        $menu = new Menu(basename(__FILE__));
-    }
-    ?>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css">
-    <link rel="stylesheet" href='<?php echo $menu->paths['css'] . "/main.css"; ?>'>
-
-</head>
-<body>
-    <?php
-    $menu->render_header();
-    $menu->render_menu();
-    ?>
-    <div class="main_content">
-
-    <div class="row">
-  <div class="column">
-    <div class="card">
-    <img src="../asset/images/aastha.jpg" alt="Avatar" style="width:100%">
-      <h3>Aastha Shrivastava</h3>
-      <p>Drupal Frontend Intern</p>
+$new = new Page;
+$content = <<<EOT
+<div class="main_content">
+  <div class="row">
+    <div class="column">
+      <div class="card">
+        <img
+          src="../asset/images/aastha.jpg"
+          alt="Avatar"
+          style="width: 100%"
+        />
+        <h3>Aastha Shrivastava</h3>
+        <p>Drupal Intern</p>
+      </div>
```

Comment:
In a followup try to refactor this in a card function, which will take a set of input and as an output will generate this card.

Reply:
YES! In some other PR we will have some function which will print books and user data and ......!